pyaedt/hfss.py

```diff
@@ -190,12 +191,22 @@ def field_setups(self):
 
         Returns
         -------
-        List of :class:`pyaedt.modules.Boundary.FarFieldSetup`
+        List of :class:`pyaedt.modules.Boundary.FarFieldSetup` and :class:`pyaedt.modules.Boundary.NearFieldSetup`
         """
         if not self._field_setups:
             self._field_setups = self._get_rad_fields()
         return self._field_setups
 
+    @pyaedt_function_handler()
+    def field_setup_names(self):
```

Comment:
You can use a property here.
```suggestion
    @property
    def field_setup_names(self):
```